dispatch/dispatcher/dispatcher.go

```diff
@@ -12,17 +12,31 @@ import (
 
 	"github.com/golang/protobuf/proto"
 
+	"github.com/iotexproject/iotex-core/blockchain"
 	"github.com/iotexproject/iotex-core/pkg/lifecycle"
+	pb "github.com/iotexproject/iotex-core/proto"
 )
 
+// Subscriber is the dispatcher subscriber interface
```

Comment:
can we merge this package back with `dispatch` now?

Reply:
Yes, will do it